Assets/Scripts/Gameplay/Level1/ChurchEvent.cs

```diff
@@ -1,28 +1,29 @@
 using UnityEngine;
-using Cinemachine;
 
 namespace ProjectWendigo
 {
     public class ChurchEvent : MonoBehaviour
     {
+
         private void Awake()
         {
             Singletons.Main.Event.On("ChurchEvent", this.EnterEvent);
         }
 
+        void Start()
+        {
+            Singletons.Main.Event.Trigger("ChurchEvent");
+        }
+
         public void EnterEvent()
         {
             this.ChangeSensitivity();
+            this.GetComponent<NearingBreathing>().Trigger();
         }
 
         public void ChangeSensitivity(float sensitivity = 0.005f)
-        { // make issue to remind self about better implementation in optionsmanager
-            // Call OptionsManager Sensitivity function once for each axis.
-
-            // Optionsmanager Sensitivity functions will look like the below...
-
-            // this.gameObject.GetComponent<CinemachineVirtualCamera>().GetCinemachineComponent<CinemachinePOV>().m_VerticalAxis.m_MaxSpeed = sensitivity;
-            // this.gameObject.GetComponent<CinemachineVirtualCamera>().GetCinemachineComponent<CinemachinePOV>().m_HorizontalAxis.m_MaxSpeed = sensitivity;
+        {
+            Singletons.Main.Option.ChangeSensitivity(sensitivity, sensitivity);
```

Comment:
Make it back to the default sensitivity after a delay.
`NearingBreathing` trigger lasts for 3 seconds by default. That time is a private property. Either make is public and use that, or use a more global time setting for the whole event, and pass it as a parameter to the `Trigger` method.
After that delay, change the sensitivity back to its default.